server/src/main/java/io/deephaven/server/config/ServerConfig.java

```diff
@@ -165,6 +171,14 @@ default int maxInboundMessageSize() {
         return DEFAULT_MAX_INBOUND_MESSAGE_SIZE_MiB * 1024 * 1024;
     }
 
+    /**
+     * How many milliseconds do we wait to shutdown the server. Defaults to {@value DEFAULT_SHUTDOWN_TIMEOUT_MILLIS}.
+     */
+    @Default
+    default int shutdownTimeoutMillis() {
+        return DEFAULT_SHUTDOWN_TIMEOUT_MILLIS;
+    }
```

Comment:
Ideally, this would be `Duration shutdownTimeout()` like `Duration tokenExpire()`. Unlike `tokenExpire()` (and side-note for myself, we probably _should_ change `tokenExpire()` to be `@Default`, but not necessary ATM), it should be a `@Default` like you have here.